submissions/usides/js-dom/dist/js/main.js

```diff
@@ -0,0 +1,197 @@
+    portret: 'crim_02.jpg',
+    files: ['crim_02_e01.jpg', 'crim_02_e02.jpg'],
+  },
+  {
+    number: 404,
+    nick: 'Night',
+    gender: 'Molly',
+    birth: '13.07.2017',
+    nationality: 'American',
+    marks: 'black as night',
+    length: 50,
+    weight: 4,
+    color: 'Black',
+    details:
+      '- Language: meowing, bip-bip-bup-bup <br>- Professional hacker and techno maniac<br>- NFC and WiFi module inside',
+    charges:
+      '- Suspected in malware production<br>- Debit and Credit Card Frauds<br>- Identity Thefts<br>- Phishing scams<br>- Hacking into bank accounts<br>- Vacuum cleaner robots hacking',
+    portret: 'crim_03.jpg',
+    files: ['crim_03_e01.jpg', 'crim_03_e02.jpg', 'crim_03_e03.jpg'],
+  },
+  {
+    number: 123,
+    nick: 'Chill',
+    gender: 'Tom',
+    birth: '01.01.2013',
+    nationality: 'American',
+    marks: 'Fluffy paws',
+    length: 45,
+    weight: 4,
+    color: 'Ginger',
+    details:
+      '- Language: meowing <br>- relaxed all the time <br>- not good at hiding',
+    charges:
+      '- Suspected in tax frauds <br>- Owner of Illegal flip flops market',
+    portret: 'crim_04.jpg',
+    files: ['crim_04_e01.jpg', 'crim_04_e02.jpg'],
+  },
+  {
+    number: 666,
+    nick: 'Katrin',
+    gender: 'Molly',
+    birth: '02.06.2014',
+    nationality: 'Russian',
+    marks: 'Sharp claws',
+    length: 50,
+    weight: 4,
+    color: 'Grey',
+    details:
+      '- Language: meowing, barking <br>- flip flops collector <br>- Charge with extreme caution',
+    charges:
+      '- Suspected in an assault with fangs and claws <br>- Terrorizes all dogs around ',
+    portret: 'crim_05.jpg',
+    files: ['crim_05_e01.jpg', 'crim_05_e02.jpg', 'crim_05_e03.jpg'],
+  },
+  {
+    number: 334,
+    nick: 'Sharpei',
+    gender: 'Tom',
+    birth: '01.03.2015',
+    nationality: 'American',
+    marks: 'Super cute',
+    length: 55,
+    weight: 4,
+    color: 'White / grey',
+    details:
+      '- Language: meowing <br>- calls himself as "sharp eye" but for some reason, other calls him as dog breed "shar-pei" <br>- master of disguise as turkey',
+    charges:
+      '- Suspected in mouses assassination with a rifle <br>- mouse killer for money - hitcat',
+    portret: 'crim_06.jpg',
+    files: ['crim_06_e01.jpg', 'crim_06_e02.jpg'],
+  },
+];
+
+const sideNavMenu = document.getElementById('side-nav-menu');
+const sideMenu = document.getElementById('side-nav');
+const menuBtn = document.getElementById('menu-btn');
+const nextBtn = document.getElementById('next-btn');
+let current = 0;
+
+document.addEventListener('DOMContentLoaded', function () {
+  loadMenu();
+  printData(current);
+});
+
+menuBtn.addEventListener('click', function (e) {
+  e.preventDefault();
+  sideMenu.classList.toggle('hidden');
+  this.classList.toggle('active');
```

Comment:
You can destructure your `e` (event) as a parameter in this fucntion and use `target` here. Maybe it will be more readable.
```suggestion
  target.classList.toggle('active');
```